### DISCUSSION TITLE
Is there a proper way to stop a Strimzi MM2 (or MM) cluster for a later start?

### DISCUSSION THREAD

**Discussion by sebady** [strimzi/strimzi-kafka-operator#6381] [Q&A] [answered]
We have some Strimzi MM/MM2 clusters that are only used during deployments.   Is there a proper way to stop a Strimzi MM/MM2 cluster to facilitate an easy restart later?

**Comment by scholzj** [comment, marked as answer]
What does it mean that they are used only during the deployments?

**Reply by sebady** [reply]
During our application's deployment we startup a Strimzi MM cluster to warmup our application before we switch processing to it.  These app deployment occur every few weeks and we'd prefer to not incur the cloud costs, while we are not using this pipeline so we've been "shutting down" MM by deleting the k8s namespace for MM.

**Reply by scholzj** [reply]
Ok. I'm still not sure I really understand your use-case. But I guess you know what you are doing. I think you have couple options:
* If you delete only the KafkaMirrorMaker2 resource, it should delete the deployment and related resources, but not the data in the Kafka clusters. So you should be able to recreate it
* You should be able to scale it to 0 (set `replicas: 0` in the KafkaMirrorMaker2 resource)

That should both work form Strimzi perspective. I think some data such as the committed offsets etc. might expire when not updated for a long time. But not sure if that would affect MM2 or your use-case or not.

**Reply by sebady** [reply]
ok.  Thank you.  I will give that a try.
